webapp/src/containers/components/modals/configModal.tsx

```diff
@@ -379,37 +393,47 @@ function ConfigModal(props: Props) {
-                    {configVisible && <Button
-                        variant='secondary'
-                        onClick={handleClose}
-                                      >{'Close'}
-                    </Button>}
-                    {actionVisible && <Button
-                        variant='secondary'
-                        onClick={handleClose}
-                                      >{'Cancel'}
-                    </Button>}
-                    {deleteVisible && <Button
-                        variant='secondary'
-                        onClick={handleClose}
-                                      >{'Cancel'}
-                    </Button>}
-
-                    {configVisible && <Button
-                        variant='primary'
-                        onClick={handleClose}
-                                      >{'Save changes'}
-                    </Button>}
-                    {actionVisible && <Button
-                        variant='primary'
-                        onClick={handleClose}
-                                      >{'Add action'}
-                    </Button>}
-                    {deleteVisible && <Button
-                        variant='danger'
-                        onClick={handleClose}
-                                      >{'Delete action'}
-                    </Button>}
+                    {configVisible &&
+                        <Button
+                            variant='secondary'
+                            onClick={handleClose}
+                        >
+                            {'Close'}
+                        </Button>}
+                    {actionVisible &&
+                        <Button
+                            variant='secondary'
+                            onClick={handleClose}
+                        >
+                            {'Cancel'}
+                        </Button>}
+                    {deleteVisible &&
+                        <Button
+                            variant='secondary'
+                            onClick={handleClose}
+                        >
+                            {'Cancel'}
+                        </Button>}
+                    {configVisible &&
+                        <Button
+                            variant='primary'
+                            onClick={handleClose}
+                        >
+                            {'Save changes'}
+                        </Button>}
+                    {actionVisible &&
+                        <Button
+                            variant='primary'
+                            onClick={handleClose}
+                        >{'Add action'}
+                        </Button>}
+                    {deleteVisible &&
+                        <Button
+                            variant='danger'
+                            onClick={handleClose}
+                        >
+                            {'Delete action'}
+                        </Button>}
```

Comment:
```suggestion
    {configVisible && handleCloseButton('primary', 'Save changes')}
    {actionVisible && handleCloseButton('primary', 'Add action')}
    {configVisible && handleCloseButton('secondary', 'Close')}
    {actionVisible && handleCloseButton('secondary', 'Cancel')}
    {deleteVisible && handleCloseButton('secondary', 'Cancel')}
    {deleteVisible && handleCloseButton('danger', 'Delete action')}
```
and create this function:
```
const handleCloseButton = (
  variant,
  text
) => (
  <Button variant={variant} onClick={handleClose}>
    {text}
  </Button>
);
```